### PR TITLE
PA-432-RemoveIdsFromQuerystring

### PR DESCRIPTION
**Pull request by Chazzysb**
PA-432-RemoveIdsFromQuerystring

### CODE REVIEW

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/AssignRequest.cs [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
 {
-    public class AssignRequest : ConfirmProviderRequest
+    public class AssignRequest : BaseAssignRequest
     {
+        public long AccountLegalEntityId { get; set; }
+    }
+
+    public class BaseAssignRequest : IndexRequest
+    {
+        public long ProviderId { get; set; }
```
Comment (benagain) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/AssignRequest.cs:13]: I think `ProviderId` should be hidden from the URL as well, shouldn't it?

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/ConfirmProviderRequest.cs [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
 {
-    public class ConfirmProviderRequest : SelectProviderRequest
+    public class ConfirmProviderRequest : BaseConfirmProviderRequest
+    {
+        public long AccountLegalEntityId { get; set; }
+    }
+
+    public class BaseConfirmProviderRequest : IndexRequest
     {
         public long ProviderId { get; set; }
```
Comment (benagain) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/ConfirmProviderRequest.cs:13]: Again, I think ProviderId should be hidden from the URL as well.

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/AssignRequest.cs (hunk 1 above)
Comment (Chazzysb) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/AssignRequest.cs:13]: provider id is entered by the user i the journey

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/ConfirmProviderRequest.cs (hunk 2 above)
Comment (Chazzysb) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/ConfirmProviderRequest.cs:13]: provider id is entered by the user in the journey

**Review by Pauljgraham — APPROVED** [review]

**Pull request merged**